### PR TITLE
zh-cn: init the translation of guidelines about issues

### PR DESCRIPTION
**Pull request by yin1999**
### Description

initial the translation of guidelines about issues

### Related issues and pull requests

~Blocked by: mdn/content#28992~

### CODE REVIEW

File: files/zh-cn/mdn/community/issues/index.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+如果你发现了错误（无论是网站的界面外观还是文档错误），你可以尝试自行修复。请阅读我们的[贡献指南](https://github.com/mdn/translated-content/blob/main/CONTRIBUTING.md)，以了解如何做出贡献。
+
+如果错误很小，例如拼写错误或小的语句改进，或者涉及到一个无争议的修复，请直接提交拉取请求以进行更改。
+
+对于所有其他类型的错误，请先[创建议题](#报告议题指南)。添加有关你打算解决该议题的意图的评论，如果可能，请描述你提议的解决方案或解决议题的步骤。等待议题被分类，以便 MDN Web 文档团队可以验证该议题是否正当，并批准你提议的解决方案。
+
+> **备注：** 如果你在对问题被分类前创建了拉取请求，那么如果链接的问题被视为无效或解决方案与 MDN Web 文档团队期望的不符，你的时间和精力可能会被浪费。请在议题被分类后，将议题分配给你自己。
+
+遵循[解决议题的指南](#解决议题的指南)，尝试通过更新适当的源内容来解决问题，例如：
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/issues/index.md:125]:
```suggestion
遵循[解决议题的指南](#解决议题指南)，尝试通过更新适当的源内容来解决问题，例如：
```

File: files/zh-cn/mdn/community/issues/index.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+### 一般分类任务
+
+- 在议题被创建后，其会被自动设置 `needs triage` 标签。你可以搜索该标签来查找需要[进行分类](#议题特定分类任务)的议题。在议题被分类之前，贡献者或其他任何人都不应该处理该议题。（分类人员请记得在对议题进行分类后删除 `needs triage` 标签。）
+- 在 [mdn/content 仓库](https://github.com/mdn/content/issues)中，还会自动设置额外的 `Content:` 标签，例如 `Content:CSS` 或 `Content:WebAPI`。这是根据议题中提到的 MDN URL 设置的。你可以使用特定内容的标签来查找需要在你的特定主题领域中进行分类的议题。
+- 如果议题涉及的活动并不是 en-US 区域的，请设置适当的标签（例如，`l10n-fr`、`l10n-zh` 或 `l10n-ja`）。这些区域的团队将会接手这些议题并对其进行分类。
+- 你不需要一直主动对议题进行分类。每周抽出一些时间（例如 30 分钟）定期对你负责的议题进行分类。分类不一定要作为同步会议的一部分进行，甚至不需要与其他人同时进行，但应定期进行，以确保未分类错误的积压不会太多。
+- 除了每周对新增的议题进行分类外，还应该查看旧议题列表，以查看是否有任何议题处于停滞状态、需要关闭或不再相关。`idle` 标签会自动设置到 30 天内没有活动的议题上。
+  - 检查仍处于打开状态的已分配议题，以检查受理人（assignee）是否正在进行处理。如果受理人在被分配后一周内没有任何进展，请询问他们是否仍有时间来解决该议题。如果又过了一周而没有任何进展，请取消分配并留下评论，说明你可以将该议题提供给其他感兴趣的贡献者。
+  - 如果已创建用于解决议题的拉取请求，但是在一周内未得到审员，请有礼貌地提醒审查员，询问他们是否可以处理该请求。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/issues/index.md:147]:
```suggestion
  - 如果已创建用于解决议题的拉取请求，但是在一周内未得到审查，请有礼貌地提醒审查员，询问他们是否可以处理该请求。
```

File: files/zh-cn/mdn/community/issues/index.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+请根据议题的严重程度为每一个错误设置优先级标签，以帮助那些想要解决最重要的议题或领域的人。
+
+- 严重议题：此类议题需要尽快修复，无论它出现在网站的哪个位置。此类议题可能会严重损害 MDN 的声誉和/或损害用户。此类议题的示例包括错误的代码片段，如果在生产中使用，可能会产生严重的安全问题和不受欢迎的内容，例如恶意软件、亵渎、色情、仇恨言论或指向此类内容的链接。
+
+  - 标签：`p0`（将立即得到解决）
+
+- 主要议题：此类议题可能会严重影响页面的实用性。例如，大量过时的信息、不起作用的复杂而重要的代码示例、大量糟糕且难以理解的行文，或大量损坏的链接。
+
+  - 标签：`p1`（将很快得到解决）和 `p2`（将很快得到解决，但优先级更改的项目将优先得到解决）
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/issues/index.md:184]:
```suggestion
  - 标签：`p1`（将很快得到解决）和 `p2`（将很快得到解决，但优先级更高的项目将优先得到解决）
```

File: files/zh-cn/mdn/community/issues/index.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+```
+
+#### 设置其他标签
+
+接下来，请根据需要设置以下标签：
+
+- `effort: small`、`effort: medium`、`effort: large`：一些贡献者喜欢根据修复错误所需的时间和精力来搜索错误。因此，如果可能的话，你应该尝试提供所需精力的估计。
+- `good first issue`：如果议题的修复非常简单，并且修复议题将为刚开始熟悉贡献流程的初学者提供良好的练习，请在议题上设置此标签。
+- `help wanted`：如果议题需要来自了解或熟悉该主题的人的帮助，请设置此标签。这是一个流行的标签，一些贡献者使用它来搜索他们熟悉或精通的开源项目中的议题。
+- `broken link internal`、`broken link external`：如果议题涉及到指向内部或外部页面的损坏链接，请设置适当的标签。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/issues/index.md:213]: Currently there is only `broken link external` label in mdn-content repo, should we remove mention of `broken link internal` 🤔 

image

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/mdn/community/issues/index.md (hunk 4 above)
Comment (yin1999) [reply files/zh-cn/mdn/community/issues/index.md:213]: the en-US content is [here](https://github.com/mdn/content/blob/7eebbf7990a48b38b1094c9ea9b6607969b34cd9/files/en-us/mdn/community/issues/index.md?plain=1#L222)

@bsmth what's your opinion on this?

File: files/zh-cn/mdn/community/issues/index.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+```
+
+#### 设置其他标签
+
+接下来，请根据需要设置以下标签：
+
+- `effort: small`、`effort: medium`、`effort: large`：一些贡献者喜欢根据修复错误所需的时间和精力来搜索错误。因此，如果可能的话，你应该尝试提供所需精力的估计。
+- `good first issue`：如果议题的修复非常简单，并且修复议题将为刚开始熟悉贡献流程的初学者提供良好的练习，请在议题上设置此标签。
+- `help wanted`：如果议题需要来自了解或熟悉该主题的人的帮助，请设置此标签。这是一个流行的标签，一些贡献者使用它来搜索他们熟悉或精通的开源项目中的议题。
+- `broken link external`：如果议题涉及到指向内部或外部页面的损坏链接，请设置此标签。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/issues/index.md:213]:
```suggestion
- `broken link external`：如果议题涉及到指向外部页面的损坏链接，请设置此标签。
```

**Review by jasonren0403 — APPROVED** [review]
Thank you @yin1999 👍

**Pull request merged**